Additional documentation (#125)
netdata/kernel-collector: docs/README.md

# Docs

> Inside this directory we are writing general documentation about this repository.

Reviewer: ```suggestion
The following directory contains general documentation related to this repository.
```
